tests/main/snap-ns-forward-compat/task.yaml

```
@@ -12,6 +12,20 @@ prepare: |
     echo "Install test snap"
     mkdir testsnap/import
     "$TESTSTOOLS"/snaps-state install-local testsnap
+    if os.query is_ubuntu_ge 25.04; then
+        # TODO remove this when it is no longer needed
```

Can you reference the issue upstream please?
```suggestion
        # FIXME: https://gitlab.com/apparmor/apparmor/-/issues/497
```